agr_literature_service/lit_processing/data_ingest/pubmed_ingest/pubmed_identify_main_pdfs.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import time
+import requests
+from agr_literature_service.lit_processing.utils.sqlalchemy_utils import \
+    create_postgres_session
+from agr_literature_service.api.models import CrossReferenceModel, ReferencefileModel
+
+logging.basicConfig(format='%(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+rootUrl = "https://www.ncbi.nlm.nih.gov"
+pmcRootUrl = 'https://ftp.ncbi.nlm.nih.gov/pub/pmc/'
+
+infile = "data/pmc_oa_files_uploaded.txt"
```

Comment:
Does this file need to be in the repo ?

Reply:
No, it is created by main PMC downloading script that calls the function in this script. This file will be cleaned up in the next run (main script is running via daily).

Reply:
Gotcha, sounds good.